app/api/models/task_comment.py

```diff
@@ -0,0 +1,29 @@
+from flask_restx import fields, Model
+
+from app.utils.enum_utils import MentorshipRelationState
+
+
+ api_namespace.models[task_comment_model.name] = task_comment_model
+ api_namespace.models[task_comments_model.name] = task_comments_model
```

Comment:
I think you are missing here the `def add_models_to_namespace(api_namespace):` line, since you are not defining the `api_namespace` anywhere.